backend/spring-routie/src/main/java/routie/place/domain/Place.java

```diff
@@ -137,13 +137,52 @@ public static Place create(
         );
     }
 
-    private static void validateStayDurationMinutes(final int stayDurationMinutes) {
+    public void modify(
+            final int stayDurationMinutes,
+            final LocalTime openAt,
+            final LocalTime closeAt,
+            final LocalTime breakStartAt,
+            final LocalTime breakEndAt,
+            final List<DayOfWeek> closedDayOfWeeks
+    ) {
+        validateForModify(stayDurationMinutes, openAt, closeAt, breakStartAt, breakEndAt);
+
+        this.stayDurationMinutes = stayDurationMinutes;
+        this.openAt = openAt;
+        this.closeAt = closeAt;
+        this.breakStartAt = breakStartAt;
+        this.breakEndAt = breakEndAt;
+        this.placeClosedDayOfWeeks.clear();
+        this.placeClosedDayOfWeeks.addAll(createClosedDayOfWeeks(closedDayOfWeeks));
+    }
+
+    private static List<PlaceClosedDayOfWeek> createClosedDayOfWeeks(final List<DayOfWeek> closedDayOfWeeks) {
+        if (closedDayOfWeeks == null || closedDayOfWeeks.isEmpty()) {
+            return new ArrayList<>();
+        }
+        return closedDayOfWeeks.stream()
+                .map(PlaceClosedDayOfWeek::new)
+                .toList();
+    }
```

Comment:
closedDayOfWeeks.isEmpty()는 검사하지 않아도 될 것 같습니다.

```suggestion
    private static List<PlaceClosedDayOfWeek> createClosedDayOfWeeks(final List<DayOfWeek> closedDayOfWeeks) {
        if (closedDayOfWeeks == null) {
            return new ArrayList<>();
        }
        return closedDayOfWeeks.stream()
                .map(PlaceClosedDayOfWeek::new)
                .toList();
    }
```

Reply:
인정합니다 👍